### PR TITLE
Misprints

### PR DESCRIPTION
**Pull request by frederikhors**

### CODE REVIEW

File: design.md [hunk 1]
```diff
@@ -209,11 +209,11 @@ We believe the Replicache model for dealing with conflicts — to have defensive
 
 It is possible for users of the Diff Server to encrypt data such that the operators of Diff Server cannot read it. Since the Client View is a key/value store, users could encrypt the values using a key that Diff Server doesn't have, but which clients and the Data Layer do.
 
-If done in a simple way, this could reduce the granularity (increase the size) of the diffs that Diff Server could produce, but there is nothing stopping users from being arbitrarily fancy with the granularity at which they encrypt data. In practive, we think a good starting point for users interested in this option is to encrypt at the value level and accept the smallish resulting increase in diff sizes.
+If done in a simple way, this could reduce the granularity (increase the size) of the diffs that Diff Server could produce, but there is nothing stopping users from being arbitrarily fancy with the granularity at which they encrypt data. In practice, we think a good starting point for users interested in this option is to encrypt at the value level and accept the smallish resulting increase in diff sizes.
```
Comment (aboodman) [inline design.md:212]: Thanks!

File: design.md [hunk 2]
```diff
@@ -209,11 +209,11 @@ We believe the Replicache model for dealing with conflicts — to have defensive
...
 
 # Constraints
 
-**Data size** A primary constraint is the size of user data. In fetching all a user's data from the data layer during each pull, Replicache makes an explicit tradeoff of bandwidth for ease of implementation and integration. For this reason we are initially limiting user data to 20MB per user and recommend the Diff Server be deployed as close to the data layer as possible (e.g., in the same Availability Zone).
+**Data size** A primary constraint is the size of user data. In fetching all user's data from the data layer during each pull, Replicache makes an explicit tradeoff of bandwidth for ease of implementation and integration. For this reason we are initially limiting user data to 20MB per user and recommend the Diff Server be deployed as close to the data layer as possible (e.g., in the same Availability Zone).
```
Comment (aboodman) [inline design.md]: I think the previous version was correct.

Comment (frederikhors) [reply design.md]: Fixed.

**Comment by aboodman** [comment]
Thank you! I realize this is somewhat silly for such a tiny change, but for cleanliness of code ownership, can you please sign our contributing agreement:

https://github.com/rocicorp/replicache/blob/master/contributing.md#legal

😀

**Comment by frederikhors** [comment]
> Thank you! I realize this is somewhat silly for such a tiny change, but for cleanliness of code ownership, can you please sign our contributing agreement:
> 
> https://github.com/rocicorp/replicache/blob/master/contributing.md#legal
> 
> 😀

Done.

**Review by aboodman — APPROVED** [review]

**Pull request merged**